packages/lesswrong/server/tableOfContents.ts

```diff
@@ -41,6 +41,52 @@ async function getTocCommentsServer (document: DbPost) {
   return getTocComments({post: document, commentCount})
 }
 
+async function getHtmlWithContributorAnnotations({
+  document,
+  collectionName,
+  fieldName,
+  version,
+  context,
+}: {
+  document: any,
+  collectionName: CollectionNameString,
+  fieldName: string,
```

Comment:
We should probably validate that the `fieldName` passed in here is in `editableCollectionsFields[collectionName]`